includes/functions/llms.functions.log.php

```diff
@@ -131,17 +131,17 @@ function llms_get_callable_name( $callable ) {
 
 	if ( is_array( $callable ) && ! empty( $callable ) ) {
 
-		// Class and class method: [ $class, 'method' ].
+		// Class and class method: [ $class, 'method' ]. (phpcs:ignore Squiz.PHP.CommentedOutCode.Found).
 		if ( is_object( $callable[0] ) ) {
 			return get_class( $callable[0] ) . '->' . $callable[1];
 		}
 
-		// Static class + method: [ 'class', 'method' ].
+		// Static class + method: [ 'class', 'method' ]. (phpcs:ignore Squiz.PHP.CommentedOutCode.Found).
 		return implode( '::', $callable );
 
 	}
 
-	// Invokable class: $class.
+	// Invokable class: $class. (phpcs:ignore Squiz.PHP.CommentedOutCode.Found).
```

Comment:
```suggestion
	// An invokable class object: $class.
```
Changes the percentage of tokens that look like code from 43% to 20%, which is under the default 40% threshold.